packages/types/Calendar.d.ts

```diff
@@ -123,9 +123,13 @@ export interface IntegrationCalendar extends Ensure<Partial<SelectedCalendar>, "
 export interface Calendar {
   createEvent(event: CalendarEvent): Promise<NewCalendarEventType>;
 
-  updateEvent(uid: string, event: CalendarEvent): Promise<Event | Event[]>;
+  updateEvent(
+    uid: string,
+    event: CalendarEvent,
+    externalCalendarId?: string | null
```

Comment:
For all calendar integrations extneral calendar id is an option field when updating bookings